<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>342</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Zero_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="One_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Two_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>350</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="three_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Dot_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>420</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Equal_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Six_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Four_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Five_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>280</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Nine_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Seven_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Eight_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Plus_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Minus_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>280</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Multip_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>210</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Division_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Percent_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Negative_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Ac_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Ac</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
